presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/ScalarSubqueryToJoinRewriter.java

```diff
@@ -50,7 +50,7 @@
 import static java.util.Objects.requireNonNull;
 
 // TODO: move this class to TransformCorrelatedScalarAggregationToJoin when old optimizer is gone
```

Comment:
Update the todo.  it seems like you want to continue to use some of this as a utility class for other subquery rewrites